app/controllers/subscribers_from_account_token_controller.rb

```diff
@@ -0,0 +1,79 @@
+class SubscribersFromAccountTokenController < ApplicationController
+  before_action :validate_params
+
+  class AccountsDown < RuntimeError; end
+  class TokenInvalid < RuntimeError; end
+
+  rescue_from AccountsDown do |exception|
+    render json: { error: "Could not fetch subscriber",
+                   details: exception.message },
+           status: :internal_server_error
+  end
+
+  rescue_from TokenInvalid do |exception|
+    render json: { error: "Could not fetch subscriber",
+                   details: exception.message },
+           status: :forbidden
+  end
+
+  def from_token
+    address = expected_params.require(:address)
+    token = expected_params.require(:token)
+
+    validate_token!(address, token)
+
+    subscriber = find_subscriber(address)
+
+    render json: { subscriber: subscriber }
+  end
+
+private
+
+  def authorise
+    authorise_user!("manage_subscriptions_from_account")
+  end
+
+  def validate_token!(address, token)
```

Comment:
Minor, this isn't a conventional use of a bang method: https://rubystyle.guide/#dangerous-method-bang